### ISSUE TITLE
clicking on the link doesn't jump to the relevant content

### ISSUE THREAD

**Issue by disneyxder** [toxoxi/remnote-table-of-contents#1]

**Comment by toxoxi** [comment]
@disneyxder 
Thank you for this report. I'll check it.

**Comment by toxoxi** [comment]
NOTE:
- this is the problem around the editor.selectRem API
  - editor.selectRem does not work when an other rem is selected
- my workaround with selectText to avoid the problem does not work currently
  - it occurs this issue
- I'm asking the behavior in the plugin channel in discord

**Comment by sotiwin** [comment]
Have you tried this way for jumping to the selected Rem:
```
    let rem = await plugin.rem.findOne(remId)
    await rem?.openRemInContext()
```

I don't have much experience in Remnote plugin development, but I managed to try this and it seems working.

**Comment by toxoxi** [comment]
@sotiwin 
Awesome! It worked well :)
https://github.com/toxoxi/remnote-table-of-contents/commit/dcea320b43f7ca6494d7454825ed3ab78b5c7265

**Comment by toxoxi** [comment]
I uploaded and it would be released after approval.

**Comment by toxoxi** [comment]
The fixed version (v 1.0.0) is released. 

@sotiwin
Thank you so much!
